src/common/uniwin.h

```diff
@@ -0,0 +1,62 @@
+#if !defined(_UNISTD_H) && defined(_MSC_VER)
+#define _UNISTD_H 1
+
+/* This is intended as a drop-in replacement for unistd.h on Windows.
+ * Please add functionality as neeeded.
+ * https://stackoverflow.com/a/826027/1202830
+ */
+
+#include <stdlib.h>
+#include <io.h>
+#include "getoptwin.h"
+#include <process.h> /* for getpid() and the exec..() family */
+#include <direct.h>  /* for _getcwd() and _chdir() */
+
+#define srandom srand
```

Comment:
srandom, random, R_OK, and many of other defines are not used anywhere in the codebase, so we probably should remove them.